WooCommerce/Classes/ViewRelated/Dashboard/Settings/Settings/SettingsViewController.swift

```diff
@@ -227,6 +222,49 @@ private extension SettingsViewController {
     }
 }
 
+// MARK: - Crash Debug Menu
+//
+extension SettingsViewController {
```

Comment:
nit: I think we can mark this extension as `private` and remove it from the functions below, it works for `@objc` function too
```suggestion
private extension SettingsViewController {
```